apps/qtmozembed/declarativewebpage.cpp

```diff
@@ -301,9 +301,12 @@ void DeclarativeWebPage::forceChrome(bool forcedChrome)
 
 void DeclarativeWebPage::grabResultReady()
 {
-    if (active()) {
-        QImage image = m_grabResult->image();
-        m_grabWritter.setFuture(QtConcurrent::run(this, &DeclarativeWebPage::saveToFile, image));
+    QImage image = m_grabResult->image();
```

Comment:
Is it ok that now we always create QImage regardless of `active()`?

Reply:
The image already exists at this point, this is just taking a copy of what is effectively a shared pointer to the actual image data.

Reply:
Ok.